x/mongo/driver/auth/creds/azurecreds.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) MongoDB, Inc. 2023-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package creds
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"go.mongodb.org/mongo-driver/internal/aws/credentials"
```

Comment:
Suggest adding a comment in `internal/aws/credentials.go` to note it is now also used to represent Azure credentials:

```
// Also used to fetch Azure credentials from the environment. Azure credentials only consist of an access token. The access token is stored in the `SessionToken` field.
```